ui/PreviewPopup.qml

```diff
@@ -194,6 +196,9 @@ Dialog {
                 rotation = 0
                 scale = 1
             }
+            onStatusChanged: {
```

Comment:
I still think this shouldn't be there at all. Manually assigning to properties (=) that were previously bound (:) tends to break the binding, which is why it's better to avoid using assignments to bound properties, generally.
Also, in this case this is just a replication of the condition that's already present in the binding.